libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/internal/kapt/KaptWithoutKotlincTask.kt

```diff
@@ -82,7 +82,11 @@ open class KaptWithoutKotlincTask @Inject constructor(private val workerExecutor
         val kaptClasspath = kaptJars + findKotlinStdlibClasspath(project)
 
         workerExecutor.submit(KaptExecution::class.java) { config ->
-            config.isolationMode = IsolationMode.PROCESS
```

Comment:
Not critical, but:
Maybe use case-insensitive comparison instead of hard-coding several possible combinations?

Reply:
Done. Thanks for the review!